node/cn/tracers/api.go

```diff
@@ -876,6 +876,9 @@ func (api *API) TraceCall(ctx context.Context, args klaytnapi.CallArgs, blockNrO
 		return nil, err
 	}
 
+	// Add gas fee to sender for estimating gasLimit/computing cost or calling a function by insufficient balance sender.
+	statedb.AddBalance(msg.ValidatedSender(), new(big.Int).Mul(new(big.Int).SetUint64(msg.Gas()), basefee))
```

Comment:
For some of chains that did not configure the KIP71 hardfork, Is there no need to update `basefee` to unit price? In this version, the `basefee` seems to be zero, leading to nothing added to the sender's balance.

Reply:
Yes, i think so too. I think it's because ethereum gasPrice policy was different from us.
before basefee,
* ethereum set gasPrice as an user argument gasPrice.
* klaytn uses unitPrice which is network-defined value. it rejects the txs which doesn't have unitPrice gasPrice.

However, I have to search the git history a bit more. What about doing this in a new PR after further discussion?

Reply:
No problem at all.

Reply:
Let's fix for chains before KIP-71.
```go
        basefee := new(big.Int).SetUint64(params.ZeroBaseFee)
        if block.Header().BaseFee != nil {
                basefee = block.Header().BaseFee
        } else {
                basefee = api.backend.ChainConfig().UnitPrice
        }
```